includes/forms/template.php

```diff
@@ -1819,7 +1819,7 @@ function give_terms_agreement( $form_id ) {
 	<?php
 }
 
-add_action( 'give_donation_form_after_cc_form', 'give_terms_agreement', 8888, 1 );
+add_action( 'give_donation_form_after_cc_form', 'give_terms_agreement', 9, 1 );
```

Comment:
@henryholtgeerts Changing hook priority here can cause unexpected results or conflict with other addons like FFM. I will suggest resolving this issue in `Give stripe` addon.

![image](https://user-images.githubusercontent.com/1784821/86007107-963f0680-ba34-11ea-8d05-7fcf8692e76f.png)